src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnectionX.cs

```diff
@@ -0,0 +1,146 @@
+
+
+        /// <inheritdoc/>
+        public override void ChangeDatabase(string databaseName)
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+        public override Task ChangeDatabaseAsync(string databaseName, CancellationToken cancellationToken = default) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+
+        public object Clone()
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <inheritdoc/>
+        public override void Close()
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+        public override Task CloseAsync() { throw new NotImplementedException(); }
+
+
+        /// <inheritdoc/>
+        public override ValueTask DisposeAsync()
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+        public override void EnlistTransaction(Transaction? transaction) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override DataTable GetSchema() { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override DataTable GetSchema(string collectionName, string?[] restrictionValues) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override DataTable GetSchema(string collectionName) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override Task<DataTable> GetSchemaAsync(CancellationToken cancellationToken = default) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override Task<DataTable> GetSchemaAsync(string collectionName, CancellationToken cancellationToken = default) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override Task<DataTable> GetSchemaAsync(string collectionName, string?[] restrictionValues, CancellationToken cancellationToken = default) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+        public override void Open()
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+
+        public override Task OpenAsync(CancellationToken cancellationToken) { throw new NotImplementedException(); }
+        /// <inheritdoc/>
+
+        protected override DbTransaction BeginDbTransaction(System.Data.IsolationLevel isolationLevel)
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+        protected override ValueTask<DbTransaction> BeginDbTransactionAsync(System.Data.IsolationLevel isolationLevel, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
+        }
+        /// <inheritdoc/>
+        protected override DbBatch CreateDbBatch() { throw new NotImplementedException(); }
```

Comment:
Nit: The curly brace needs to begin on a new line according to C# coding.

Reply:
Note that you can also use expression-bodied method syntax:

```c#
protected override DbBatch CreateDbBatch()
    => throw new NotImplementedException();
```